packages/tools/cookbook/src/utils/account-key.ts

```diff
@@ -0,0 +1,5 @@
+// Returns the public key for the provided account name assuming
+// the naming follows the recommended convention: "k:publicKey"
+export function accountKey(account: string): string {
+  return account.split(':')[1];
```

Comment:
Can we do a check if this account is a namespaced account, and throw an error when it isn't. Gives a bit more info for the end user when this doesn't work (when they don't read the docs ;) )